force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +46,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        Boolean ageCheck = false;
+        if (age > 17){
+            ageCheck = true;
+        }
+        return ageCheck; // Replace null with the variable you used to store the result
```

Comment:
Good the suggestion I give to students for this one is to understand how you can solve this with one line of code. 

Think about what expressions using < or > evaluate too.